dev/CommandBarFlyout/InteractionTests/CommandBarFlyoutTests.cs

```diff
@@ -620,5 +620,48 @@ public void VerifyFlowsToAndFromIsNotSetWithoutPrimaryCommands()
                 InputHelper.Tap(showCommandBarFlyoutButton);
             }
         }
+
+        [TestMethod]
+        public void VerifyFlyoutClosingBehavior()
+        {
+            if (PlatformConfiguration.IsOSVersionLessThan(OSVersion.Redstone2))
+            {
+                Log.Warning("Test is disabled pre-RS2 because CommandBarFlyout is not supported pre-RS2");
+                return;
+            }
+
+            using (var setup = new CommandBarFlyoutTestSetupHelper())
+            {
+                Button showCommandBarFlyoutButton = FindElement.ByName<Button>("Show CommandBarFlyout with sub-menu");
+
+                Log.Comment("Tapping on a button to show the CommandBarFlyout.");
+                InputHelper.Tap(showCommandBarFlyoutButton);
+
+                // Pre-RS5, CommandBarFlyouts always open expanded,
+                // so we don't need to tap on the more button in that case.
+                if (PlatformConfiguration.IsOsVersionGreaterThanOrEqual(OSVersion.Redstone5))
+                {
+                    Log.Comment("Expanding the CommandBar by invoking the more button.");
+                    FindElement.ById<Button>("MoreButton").InvokeAndWait();
+                }
+
+                // Move around over item to open flyout
+                PointerInput.Move(FindElement.ById("ProofingButton"), 5, 5);
+                PointerInput.Move(FindElement.ById("ProofingButton"), 6, 5);
+                PointerInput.Move(FindElement.ById("ProofingButton"), 5, 6);
```

Comment:
I made a change yesterday that would allow you to replace this with a click if you wanted to

Reply:
Oh right, yes. I don't think there is much benefit in updating this, but I'll use it next time when I write a test in this area.